jenkins/build.sh

```diff
@@ -73,15 +73,15 @@ rm -rf ${BASE_DIR}/dist
 cp -rf ${BASE_DIR}/conf/etc/v2/ha.conf.template ${BASE_DIR}/conf/etc/v2/ha.conf
 sed -i -e "s|<VERSION>|${version}|g" ${BASE_DIR}/conf/etc/v2/ha.conf
 
-python3.6 setup.py bdist_rpm --version="${version}" --install-dir="${HA_INSTALL_PATH}" --release="$BUILD" \
---force-arch x86_64 --post-install jenkins/rpm/v2/k8s/post_install_script.sh \
+python3.6 setup.py bdist_rpm --version="${version}" --install-dir="${HA_INSTALL_PATH}" --binary-only --release="$BUILD" \
```

Comment:
binary-only flag will generate only binary and not source. Is that intended ? Will source files be available inside deployed container after this change ?

Reply:
>  Will source files be available inside the deployed container after this change?
Yes, I am able to see the source file inside the container after deployment.

Reply:
@ArchanaLimaye

Reply:
@shailesh-vaidya, Please confirm this comment.

Reply:
@ArchanaLimaye @satish-seagate  - We do not need source rpm. ie. .src.rpm file. We were not copying it to dist directory also. refer - https://github.com/Seagate/cortx-ha/blob/main/jenkins/build.sh#L82. Hence disabled it's generation. 
Generated rpm has all the source files.

Reply:
okay